src/main.js

```diff
@@ -26,6 +27,7 @@ const routes = [
   { path: '/login', component: LoginForm },
   { path: '/signup', component: SignupForm },
   { path: '/whiteboard', component: Whiteboard, meta: { protected: false } },
+  { path: '/messager', component: Messager, meta: {protected: false}},
```

Comment:
No need to make this its own route, instead, in Whiteboard component, you can import the Messager (which should probably be corrected to `Messenger` or `Chat`) and use it in the template directly. See Onboarding.vue as an example (or much of the recent stuff in `master`)